go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/utils
 
-go 1.18
+go 1.21.0
```

Comment:
leave this at go1.18, bumping here has side effects we don't want (https://github.com/kubernetes/kubernetes/issues/123744)

Reply:
cc @MadhavJivrajani